text/0685-new-browser-support-policy.md

```diff
@@ -0,0 +1,293 @@
+and users are generally on one of the more recent versions of a browser, this
+dynamic changes. We cannot set an explicit version to support anymore, because
+it would be impractical - Chrome alone releases a new version every 6 weeks.
+Setting an explicit lowest-supported-version would lock us into having to
+support a huge number of older browsers, many of which are entirely unused.
+
+Conversely, however, tracking the latest release of a browser may not be enough
+stability for Ember apps. If Ember were to adopt a new browser feature
+immediately after its release, even though many web users were still using a
+previous version, it would cause many Ember apps to either break, or be unable
+to update until their users had also updated to the latest browser. Even a
+rolling `N - 1` or `N - 2` support policy, where the last 1 or 2 versions are
+supported, may not be enough. Many users were stuck in the recent Edgium update
+for some time, for instance.
+
+This RFC seeks to establish a new support policy that addresses these issues
+with a new set of heuristics. These heuristics distinguish between two types of
+browsers which are supported:
+
+- **Evergreen browsers**, those which have adopted a continuous update policy
+  and generally keep their users up-to-date automatically.
+- **Non-evergreen browsers**, which have more complicated support patterns that
+  may prevent users from updating regularly.
+
+Depending on which category a browser falls into, different rules will be
+applied. This will allow us to effectively support browsers which release
+frequently, such as Chrome, and browsers which do not, such as Safari.
+
+## Existing Policy
+
+The existing policy is listed here for documentation purposes.
+
+- Desktop
+
+    1. Google Chrome, Mozilla Firefox, MS Edge, Mac Safari
+
+        - The current and previous stable releases are supported.
+
+    2. Internet Explorer
+
+        - Only Internet Explorer 11 is supported.
+
+- Mobile
+
+    1. All
+
+        - All mobile browsers may work but are not explicitly supported.
+
+Any browsers not listed here may work but are not explicitly supported.
+
+## Proposed policy
+
+In the new support policy, we define two categories of browsers:
+
+1. **Evergreen browsers**, which are browsers that:
+
+   * Support using the latest version on all supported platforms (e.g. you can
+     use the latest Chrome an any supported version of Windows, macOS, Android,
+     etc. The browser is versioned independently from the operating system).
+   * Automatically update whenever a new version is available.
+
+2. **Non-evergreen browsers**, which are any browsers which do not meet the
+   criteria of evergreen browsers. For instance, Safari is a supported browser
+   whose version is tied to the version of macOS and iOS that users use. As
+   users will often wait to update their operating system, many users lag behind
+   on the version of Safari they are using, so it cannot be considered
+   evergreen.
```

Comment:
Not sure how much it makes sense to include it here, but obligatory mention of Safari Tech Preview 😉.